### ISSUE TITLE
Error E0019 is undocumented

### ISSUE THREAD

**Issue by Michael-F-Bryan** [rust-lang/rust#43308]
I was just playing with conditions in `const` functions and came across an error code (*E0019) which isn't yet documented in the error index.

([playground](https://play.rust-lang.org/?gist=8208393c563d7c999febecfc480a8e8f&version=nightly))

---

For example, trying to compile this:

```rust
#![feature(const_fn)]
const fn foo() -> u8 {
    if bar() == 1 { 5 } else { 3 }
}

const fn bar() -> u8 {
    1
}
fn main() {}

```

Gives you

```
 Compiling playground v0.0.1 (file:///playground)
error[E0019]: constant function contains unimplemented expression type
 --> src/main.rs:3:5
  |
3 |     if bar() == 1 { 5 } else { 3 }
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `playground`.
```

Where [E0019](https://doc.rust-lang.org/error-index.html#E0019) just sends you to the top of the page because the *E0019* section doesn't exist.

---

What would be the best way for me to add an explanation of that error code to the compiler error index?

**Comment by kennytm** [comment]
This is a bug in the error index generator. E0019, along with many other error codes, [are documented](https://github.com/rust-lang/rust/blob/2652ce6771b114189cdc1f9bd026a10af0a714e0/src/librustc_mir/diagnostics.rs#L136-L170) yet not included.

Duplicate of #42170 (E0493 missing — E0493 and E0019 live in the same file), which is closed in favor of #34588 (error index generator is fragile).

**Comment by Michael-F-Bryan** [comment]
Cheers @kennytm, I'll close this in favor of #34588 as well.
